### PR TITLE
integration of socket for ping module

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
This marks the first integration of socket with ping.
Major changes.

[working model video](https://drive.google.com/file/d/1BjyLbjysLU9udCIFR9rAeeEbP0e5Hrvp/view)

### CODE REVIEW

File: src/lib/handlers.go [hunk 1]
```diff
@@ -10,17 +15,132 @@ const (
...
+	case "start":
+		if pingServiceState == "passive" {
+
+			Configuration.Config.UtilsConf.ServicesSignal.Ping = "active"
+			_, e := Configuration.Write()
+			if e != nil {
+				panic(e)
+			}
+			go func() {
+				fmt.Println("inside goroutine")
```
Comment (aquibbaig) [inline src/lib/handlers.go]: Remove Println/console logs after debugging

File: src/lib/handlers.go [hunk 2]
```diff
@@ -10,17 +15,132 @@ const (
...
+				pingConfig := Configuration.Config.Routes
+				pingInterval := getInterval(Configuration.Config.Interval, "ping")
+				if pingInterval == (testInterval{}) {
+					log.Fatalf("interval not found in configuration file for ping")
+					return
+				}
+				fmt.Println("ping configuration is")
+				fmt.Println(pingConfig)
```
Comment (aquibbaig) [inline src/lib/handlers.go]: Here too

File: src/lib/handlers.go (hunk 1 above)
Comment (Harkishen-Singh) [reply src/lib/handlers.go]: done!

File: src/lib/handlers.go (hunk 2 above)
Comment (Harkishen-Singh) [reply src/lib/handlers.go]: done

**Pull request merged**